support-workers/deploy-to-code.sh

```diff
@@ -0,0 +1,13 @@
+S3_BUCKET="membership-dist"
+S3_KEY="support/CODE/support-workers-typescript/support-workers.zip"
+LAMBDA_FUNCTIONS=(
+  "-CreatePaymentMethodLambda-"
```

Comment:
why not put the `-` where it's used rather than in (every) function name?
`support-${function}-CODE`